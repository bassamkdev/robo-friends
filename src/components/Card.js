import React from 'react';

const Card = ({name, email, id}) => {
	return (
		<div className='tc bg-light-green dib pa3 ma2 br3 bw2 shadow-5 grow'>
			<img alt ='robots' src={`https://robohash.org/${id}?200x200`}/>
			<div>
				<h2>{name}</h2>
				<p>{email}</p>
			</div>
		</div>
	);
}

export default Card;