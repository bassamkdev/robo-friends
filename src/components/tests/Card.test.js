import React from 'react';

import Card from '../Card';

import { shallow } from 'enzyme';

it('render correctly Card component', () => {
    const wrapper = shallow(<Card />);
    expect(wrapper).toMatchSnapshot();
})