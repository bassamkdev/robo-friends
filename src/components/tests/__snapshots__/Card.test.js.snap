// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly Card component 1`] = `
<div
  className="tc bg-light-green dib pa3 ma2 br3 bw2 shadow-5 grow"
>
  <img
    alt="robots"
    src="https://robohash.org/undefined?size=200x200"
  />
  <div>
    <h2 />
    <p />
  </div>
</div>
`;
